Library/Homebrew/formula_creator.rb

```diff
@@ -168,11 +168,9 @@ def install
         <% elsif mode == :go %>
             system "go", "build", *std_go_args(ldflags: "-s -w")
         <% elsif mode == :meson %>
-            mkdir "build" do
-              system "meson", *std_meson_args, ".."
-              system "ninja", "-v"
-              system "ninja", "install", "-v"
-            end
+            system "meson", "setup", "build", *std_meson_args
+            system "meson", "compile", "-C", "build", "--verbose"
```

Comment:
Note that this will simply import approximately 190 python files amounting to 70k lines of code, check whether the setup was configured with an explicit --backend option and thus doesn't use the default ninja, and potentially run either one of:
- `ninja -v`
- `xcodebuild`

It adds no capabilities over ninja, but if you prefer it...

Reply:
Mainly matching current style preference in `homebrew-core`. Similar to CMake changes, though there is probably more overhead in Meson's Python code path than CMake's C/C++. I assume it isn't going to make a noticeable impact when actually run on CI. It leaves open the possibility of tweaking our `std_meson_args` backend even if it isn't very likely change.

At least other repositories that also directly run commands in packages (e.g. Arch, Alpine) seem to favor running through Meson.

Repositories that use their own helper commands (e.g. Debian, Fedora) seem to directly run Ninja behind the scenes (e.g. Fedora's `%meson_build` being an alias for `%ninja_build`)